scripts/export_account_data.py

```
@@ -0,0 +1,569 @@
+            print(f"❌ Error retrieving app key: {e}")
+            sys.exit(1)
+    
+    def decrypt_value(self, encrypted_value: str) -> Optional[str]:
+        """Decrypt a Laravel encrypted value."""
+        if not encrypted_value or not self.app_key:
+            return encrypted_value
+        
+        try:
+            # Laravel encryption format: base64(serialize(encrypted_data))
+            # The encrypted data contains: iv + ciphertext + mac
+            encrypted_data = base64.b64decode(encrypted_value)
```

Ruff (0.11.9)

Local variable `encrypted_data` is assigned to but never used

Remove assignment to unused variable `encrypted_data`

(F841)